node/src/sub_lib/neighborhood.rs

```diff
@@ -59,12 +59,12 @@ pub struct RatePack {
 }
 
 impl RatePack {
-    pub fn routing_charge(&self, payload_len: usize) -> u64 {
-        self.routing_service_rate + (self.routing_byte_rate * payload_len as u64)
+    pub fn routing_charge(&self, payload_size: u64) -> u64 {
+        self.routing_service_rate + (self.routing_byte_rate * payload_size)
     }
 
-    pub fn exit_charge(&self, payload_len: usize) -> u64 {
-        self.exit_service_rate + (self.exit_byte_rate * payload_len as u64)
+    pub fn exit_charge(&self, payload_size: u64) -> u64 {
```

Comment:
Using `u64` adds a convenience over here. Since, any value bigger than `u64` would never be used, it makes more sense to keep the fn signature accurate instead of using vague data type like `usize`.